app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")

```

Comment:
![image](https://user-images.githubusercontent.com/104218550/198878268-87a96e1a-b229-470c-a4ef-6a3764a13e41.png)

Reply:
it is in all other similar cases